handbook/ce/customer-support-onboarding.md

```diff
@@ -1,5 +1,204 @@
+
+Our onboarding is centered on outcomes: With X time interval, we expect Y outcome. In support of this, each time interval has a task list. If you complete the tasks and you do not feel the outcomes to be true, that likely indicates something needs to be improved about the onboarding. Make note of it, share it with the team, and we will figure out how to make sure you feel the outcomes to be true and improve the checklist for the next new team member. At this point, it’s not you. This onboarding plan is new and requires so much improvement.
+
+As you complete the company onboarding checklist, let yourself really spend time getting to know the handbook. For each item on the checklist that takes you to the handbook, set a timer for 90 minutes and just let yourself click and read, click and read. For example, when you get to the item “Read handbook page on communication,” click into the sub links listed on that page and read those, too.
+
+Your first week will be very reading intensive. As you read, share your thoughts, reactions, and questions in our #customer-support-chat Slack channel. It makes it a little more interactive and less tedious.
+
+Okay, stop! Head back over to the company onboarding checklist and finish it (if you haven't done so already). Then, come back here and dig in!
+
+✨
+
+Welcome back! Let’s get started....
+
+## By the end of your first week, the following will be true...
+* You understand Sourcegraph’s mission, vision, values, and culture
+* You understand Sourcegraph’s purpose is and how you/your role supports us achieving it
+* You understand what is expected of you in your role and what it means to be a successful Customer Support Engineer
+* You have a sense of what tools you will be using and for what
+
+### In order to ensure the above declarations are true, you will complete the following tasks:
+* Week one items in the company onboarding Asana checklist.
+* Review our [FY22 OKRs](https://docs.google.com/document/d/18d3sX38O6ephNuoHqZT9BwU1m1_FGuRfsTwYbc8lMV8/edit#heading=h.obnaanj0svtb) (objectives and key results) to see what we believe are the most important things for us to do this year. These frame our decision making. When we are confronted with conflicting priorities, our first step is to ask ourselves … if this takes away our ability to complete an OKR, does that mean it’s more important than the OKR? It might be, but this gut check makes sure we are spending our time in the right way. If OKRs are new to you, Virginia can give you a crash course on what they are, their value, and how to work with them.
+* Read [this article](https://www.infoworld.com/article/3514213/sourcegraph-universal-code-search-and-intelligence.html) that summarizes the value of Sourcegraph
+* Plan to attend the weekly “Product Tour” session (on Thursdays) for your first 3-4 weeks. It will be repetitive, but each week, more and more will start to click. 
```

Comment:
I didn't read that note 🤦‍♀️ all the way through yesterday. Oooops. @mike-r-mclaughlin @emchap do either of you have a link you recommend? And for Chorus, do you need an account to watch things? If so, a YouTube link would be better.

Reply:
there's one on the CE onboarding page linked: https://www.youtube.com/watch?v=iTBTri_q5MA&feature=youtu.be

Reply:
@virginiaulrich tbh I don't know! I know you can generate links to demos that don't require auth. Would we maybe want to ask IT if they're cool with that (just since public URLs are a risk) and if so we can pick three or four?